runtime/src/tiered_storage/meta.rs

```diff
@@ -0,0 +1,76 @@
+//! The account meta and related structs for the tiered storage.
+
+use modular_bitfield::prelude::*;
+
+/// The struct that handles the account meta flags.
+#[allow(dead_code)]
+#[bitfield(bits = 32)]
+#[repr(C)]
+#[derive(Debug, Default, Copy, Clone, Eq, PartialEq)]
+pub(crate) struct AccountMetaFlags {
+    /// whether the account meta has rent epoch
+    #[allow(dead_code)]
+    has_rent_epoch: bool,
+    /// whether the account meta has account hash
+    #[allow(dead_code)]
+    has_account_hash: bool,
+    /// whether the account meta has write version
+    #[allow(dead_code)]
+    has_write_version: bool,
+    /// the reserved bits.
+    #[allow(dead_code)]
+    reserved: B29,
+}
```

Comment:
Not sure how to work mark the generated code `dead_code` as I already have `#[allow(dead_code)]` above it.

```
error: associated function `new` is never used
--
  | --> runtime/src/tiered_storage/meta.rs:5:1
  | \|
  | 5  \| // /// The struct that handles the account meta flags.
  | 6  \| \|\| #[allow(dead_code)]
  | 7  \| \|\| #[bitfield(bits = 32)]
  | 8  \| \|\| #[repr(C)]
  | 9  \| \|\| #[derive(Debug, Default, Copy, Clone, Eq, PartialEq)]
  | 10 \| \|\| pub(crate) struct AccountMetaFlags {
  | \| \|\|__________________________________- associated function in this implementation
  | ...  \|
  | 22 \| \|      reserved: B29,
  | 23 \| \|  }
  | \| \|__^
  | \|
  | = note: `-D dead-code` implied by `-D warnings`
  |  
  | error: associated items `into_bytes` and `from_bytes` are never used
  | --> runtime/src/tiered_storage/meta.rs:5:1
  | \|
  | 5  \| // /// The struct that handles the account meta flags.
  | 6  \| \|\| #[allow(dead_code)]
  | 7  \| \|\| #[bitfield(bits = 32)]
  | 8  \| \|\| #[repr(C)]
  | 9  \| \|\| #[derive(Debug, Default, Copy, Clone, Eq, PartialEq)]
  | 10 \| \|\| pub(crate) struct AccountMetaFlags {
  | \| \|\|__________________________________- associated items in this implementation
  | ...  \|
  | 22 \| \|      reserved: B29,
  | 23 \| \|  }
  | \| \|__^
  | \| \|__\|
  | \|


```

Reply:
Placing `#![allow(dead_code)]` at the top of the file does the magic.  Thank you!